intl/msg_hash_us.h

```diff
@@ -1743,6 +1847,10 @@ MSG_HASH(
    MENU_ENUM_SUBLABEL_VIDEO_REFRESH_RATE_AUTO,
    "The accurate estimated refresh rate of the screen in Hz."
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_VIDEO_REFRESH_RATE_AUTO,
+   "The accurate refresh rate of your monitor (Hz). This is used to calculate audio input rate with the formula:\naudio_input_rate = game input rate * display refresh rate / game refresh rate\nIf the implementation does not report any values, NTSC defaults will be assumed for compatibility.\nThis value should stay close to 60Hz to avoid large pitch changes. If your monitor does not run at 60Hz, or something close to it, disable VSync, and leave this at its default."
```

Comment:
Minor comment: this string will definitely overflow in other languages. Keep it in mind for the expansion issue.

Reply:
Noted, will consider it for phase 2 PR.